lib/v2/screens/dashboard/interactor/viewmodels/balance_event.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class BalanceEvent extends Equatable {
+  @override
+  List<Object> get props => [];
+}
+
+class OnBalanceUpdate extends BalanceEvent {
```

Comment:
`OnLoadBalance`
`OnRefreshBalance`

I m not much into the `update` word for naming these

Reply:
True that update has 2 meanings - I will change